README.md

```diff
@@ -39,47 +38,17 @@ For example, "500,ACT,2", where ACT is the command and 2 is the bank. Timestamp
 mentioned in [MemCommand.h](src/MemCommand.h) and the bank is the target bank number. For non-bank-specific commands, bank can be set to 0. Rank need not be
 specified. The timing correctness of the trace is not verified by the tool and is assumed to be accurate. However, warning messages are provided, to identify if the memory or bank state is inconsistent in the trace. A sample command trace is provided in the traces/ folder.
 
-### Transaction Traces
```

Comment:
I would mention in the readme with a pointer to the last commit before this PR, that transaction traces did exists before. And for future versions we will rely on simulators like DRAMSys and Ramulator for this purpose. In case somebody wants this feature he could checkout a older version.

Reply:
Sure, I will add that in Readme.

Reply:
Updated the ReadMe,